src/main/java/seedu/address/logic/commands/SessionCommand.java

```diff
@@ -0,0 +1,98 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_SESSION;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Session;
+import seedu.address.model.person.SessionList;
+
+/**
+ * Adds session to an existing person in the address book.
+ */
+public class SessionCommand extends Command {
+
+    public static final String COMMAND_WORD = "session";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Adds session to the person identified "
+            + "by the index number used in the last person listing. "
+            + "Existing session will not be modified.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "[" + PREFIX_SESSION + "SESSION]...\n"
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_SESSION + "08:30-09:30";
```

Comment:
For the session timing, let's leave it as just the day and starting time as mentioned in our user guide

Reply:
ok forgot about the day, will be sure to be done as stipulated by tomorrow hold on @AshiqurRah <3